tools/proofers/preview.js

```diff
@@ -895,12 +956,42 @@ $(function () {
             if (viewMode !== "show_tags") {
                 colorString = colorString0 + '>$1</span>';
             }
-            // sub- and super-scripts
-            txt = txt.replace(/_\{([^\}]+)\}/g, '<span class="sub"' + colorString);
-            txt = txt.replace(/\^\{([^\}]+)\}/g, '<span class="sup"' + colorString);
-            // single char superscript -  any char except {
-            // do not allow < incase it's a tag which would screw up
-            txt = txt.replace(/\^([^\{<])/g, '<span class="sup"' + colorString);
+
+            // process sub- and super-scripts
+            function ssProcess(text) {
```

Comment:
is ss common to refer to sub and super scripts? I only think of some other historical entities.

Reply:
I guess you mean this: https://en.wikipedia.org/wiki/SS_Jaguar_100.
I'll make a better name.